generators/entity/index.js

```diff
@@ -128,6 +130,12 @@ module.exports = class extends BaseGenerator {
         if (useBlueprint) return;
         return {
             getConfig() {
+                if (this.availableGeneratorConfig) {
+                    this.config = this.availableGeneratorConfig;
+                    this.config.getAll = () => this.config;
```

Comment:
why are these needed?

Reply:
to replace the seemingly absent config passing from one generator to another, but with your previous tip, I might get it to work without it

Reply:
@MathieuAA : these 5 lines made `this.config` wrong. If I deleted these 5 lines, it seems to work